contentctl/objects/config.py

```diff
@@ -249,11 +254,52 @@ class StackType(StrEnum):
     classic = auto()
     victoria = auto()
 
+
 class inspect(build):
```

Comment:
Is there a way to enable enrichments by default for the `inspect` command? The flag is strictly necessary w/ the inclusion of version enforcement, as w/o MITRE data, local builds will look much different than the release on Splunkbase and raise lots of false errors

Reply:
The validator is a great idea, but I'm second guessing myself now. On second thought, ESCU always has enrichments in its published releases, but a 3rd party user of contentctl might not in their app.

Forcing enrichments to be on makes sense for our workflow, but not for others. Thoughts?

Reply:
I’m going to make a few assumptions:

1. Even if someone doesn’t have MITRE tagging in their app, requiring enrichment to be enabled doesn’t “cost” them anything or should not cause additional failed validations.
2. If someone DOES have any tagging, they really NEED enrichment to be enabled for versioning check to work as planned.
3. Anyone advanced enough to want the versioning check is almost certainly going to have things tagged with MITRE Tactics.
4. If someone has at least 1 MITRE tag, enrichment should DEFINITELY be enabled when doing a versioning check.
Given those points, I think we should just force enrichments to be enabled if version checking is enabled. The alternative, which is “iterate over all content to see if there are any MITRE tags” I think should not be necessary.

Reply:
Yeah agreed re: the rollout of this ES8 feature being clunky

1. great point; there’s nothing to enrich if they aren’t using MITRE stuff
2. also a great point; why would they be using MITRE tags and then publishing unenriched content
3. see 2
4. see 2

So I’m in agreement w/ using a validator to enforce enrichments are on when metadata validation is :slightly_smiling_face:

Reply:
Added `field_validator` on `enrichments` for the `inspect` action

If it looks good @pyth0n1c please resolve this :)